help_popup/models/action_window.py

```diff
@@ -0,0 +1,179 @@
+# coding: utf-8
+# © 2015 David BEAL @ Akretion <david.beal@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+import inspect
+
+from openerp import _, api, models, fields
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from bs4 import BeautifulSoup as BSHTML
+except ImportError:
+    _logger.debug(
```

Comment:
I guess this could be at least a warning error, if we want to install this module I guess this import is required